.github/workflows/full-publish.yaml

```diff
@@ -0,0 +1,26 @@
+name: Publish release
+on:
+  push:
+    branches:
+      - publish
+jobs:
+  gradle:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4.2.2
+      - uses: actions/setup-java@v4.6.0
+        with:
+          distribution: "temurin"
+          java-version: "11"
```

Comment:
I'm also unsure about this part. 
I have no idea if this works or if we need some other action